### PR TITLE
Np 2617 remove hardcoded series

### PR DESCRIPTION
**Pull request by axthosarouris**

### CODE REVIEW

File: cristin-import/src/main/java/no/unit/nva/cristin/lambda/constants/HardcodedValues.java [hunk 1]
```diff
@@ -8,15 +8,18 @@
     public static final String HARDCODED_BOOK_PUBLISHER = "SomePublisher";
     public static final Level HARDCODED_LEVEL = Level.LEVEL_0;
     public static final String HARDCODED_PAGE = "1";
-    public static final String HARDCODED_SERIES_NUMBER = "1";
```
Comment (axthosarouris) [inline cristin-import/src/main/java/no/unit/nva/cristin/lambda/constants/HardcodedValues.java:11]: These fields are not obligatory and we do not need to have hardcoded values

File: cristin-import/src/main/resources/ignoredFields.txt [hunk 2]
```diff
@@ -16,6 +16,8 @@ entityDescription.contributors.email
 entityDescription.date.day
 entityDescription.date.month
 entityDescription.reference.publicationContext.isbnList
+entityDescription.reference.publicationContext.seriesNumber
```
Comment (axthosarouris) [inline cristin-import/src/main/resources/ignoredFields.txt:19]: Add the fields to the ignored-fields list

File: cristin-import/src/main/java/no/unit/nva/cristin/lambda/constants/HardcodedValues.java [hunk 3]
```diff
@@ -8,15 +8,18 @@
...
     public static final URI HARDCODED_URI = URI.create("https://www.example.com/");
     public static final String HARDCODED_NPI_SUBJECT = "1007";
     public static final URI HARDCODED_NVA_CUSTOMER =
         URI.create("https://api.dev.nva.aws.unit.no/customer/f54c8aa9-073a-46a1-8f7c-dde66c853934");
 
     public static final URI HARDCODED_SAMPLE_DOI = URI.create("https://doi.org/10.1145/1132956.1132959");
 
+    public static final boolean HARDCODED_ILLUSTRATED = false;
+    public static final boolean HARDCODED_PEER_REVIEWED = false;
+    public static final boolean HARDCODED_TEXTBOOK_CONTENT = false;
```
Comment (torarnet) [inline cristin-import/src/main/java/no/unit/nva/cristin/lambda/constants/HardcodedValues.java:21]: Did you have these in last PR as well?

**Review by torarnet — APPROVED** [review]

File: cristin-import/src/main/java/no/unit/nva/cristin/lambda/constants/HardcodedValues.java (hunk 3 above)
Comment (axthosarouris) [reply cristin-import/src/main/java/no/unit/nva/cristin/lambda/constants/HardcodedValues.java:21]: no

**Pull request merged**